compact_proposal/makefile

```diff
@@ -1,18 +1,22 @@
-all: test_proposal_final.pdf test_proposal_draft.pdf
+all: test_proposal_final.pdf test_proposal_draft.pdf test_proposal_draft_nonotes.pdf
 
 .PHONY = all open
 
 test_proposal_draft.pdf: test_proposal_draft.tex compact_proposal.cls
 	pdflatex	test_proposal_draft.tex
 	pdflatex	test_proposal_draft.tex
-	open	test_proposal_draft.pdf	
-	
+	# open	test_proposal_draft.pdf
+
+test_proposal_draft_nonotes.pdf: test_proposal_draft_nonotes.tex compact_proposal.cls
+	pdflatex	test_proposal_draft_nonotes.tex
+	pdflatex	test_proposal_draft_nonotes.tex
+
 test_proposal_final.pdf: test_proposal_final.tex compact_proposal.cls
 	pdflatex	test_proposal_final.tex
 	pdflatex	test_proposal_final.tex
-	open	test_proposal_final.pdf	
-	
-open_both: 
-	open	test_proposal_draft.pdf
-	open	test_proposal_final.pdf
-	
\ No newline at end of file
+	# open	test_proposal_final.pdf
+
+# is there a way to open PDF files in an OS-agnostic way??
```

Comment:
no, you have to make check for the OS and then do conditionals off that (as far as I know). Check out function `uname`.